botos/settings/regular_settings.py

```diff
@@ -113,6 +113,7 @@ def get_env_var(var_name, env_source=os.environ, value_meanings=None,
                          debug=DEBUG,
                          debug_value=os.path.join(BASE_DIR, 'botos/media/'))
 
+APPEND_SLASH=False
```

Comment:
Why do we need to set `APPEND_SLASH` to `False`?

Reply:
Some buttons post to paths that don't have a slash. I'll fix it once I find where the file with the paths are